drivers/storage/portworx/deployment.go

```diff
@@ -692,6 +692,11 @@ func (t *template) getSelectorTerms(k8sVersion *version.Version) []v1.NodeSelect
 			Operator: v1.NodeSelectorOpNotIn,
 			Values:   []string{"false"},
 		},
+		{
```

Comment:
This will apply the selectors to only px pods and not the rest of the components. Was this intentional?

Reply:
This will apply to all pods, as discussed over call